cpp/src/arrow/engine/substrait/options.h

```diff
@@ -23,7 +23,11 @@
 #include <string>
 #include <vector>
 
+#include <google/protobuf/any.pb.h>
```

Comment:
This would mean that libarrow_python would require protobuf headers to build.

Reply:
I'm not familiar with libarrow_python build, but it sounds like you're saying adding this would be bad. Should the APIs involving `Any` be made internal?